tilings/algorithms/minimal_gridded_perms.py

```diff
@@ -485,9 +485,15 @@ def _process_work_packet(
             # if gp was one of the initial_gps that satisfied obs/reqs, but
             # we weren't sure at the time if it was minimal, then now is the
             # time to check and yield
-            if qpacket.gp in initial_gps_to_auto_yield:
+            if curr_len != len(qpacket.gp):
+                curr_len = len(qpacket.gp)
+                for gp in initial_gps_to_auto_yield[curr_len]:
```

Comment:
you should yield from range(prev_len + 1, curr_len + 1)

Reply:
You're right. I changed it.